sdk/formrecognizer/Azure.AI.FormRecognizer/src/ExtractFormOperation.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    /// <summary>
+    /// </summary>
+    internal class ExtractFormOperation : Operation<ExtractedForm>
```

Comment:
Wonder if we can extract common logic from these operation later.

Reply:
We definitely should do it eventually, seems rather duplicated.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/10435